docs/vertiq_testing_tool/vertiq_testing_tool_start_guide.rst

```diff
@@ -0,0 +1,128 @@
+When this feature is enabled, the ESC command will be continuously sent at the 'Command rate' with 10 Hz being the default value. The 'Command rate' can be adjusted to send the ESC command at different rates. 
+You can also use the slider to adjust the ESC input value, which mimics adjusting the flight controller's throttle input between [0, 1].
+
+
+================================
+Live Plotting and Active Metrics
+================================
+
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input_command_example.png
+One main feature of Vertiq Testing Tool is the ability to plot any supported metric. 
+Please refer to the :ref:`IQUART Client Table Reference tables <iquart_client_reference_tables>` to find the endpoints that allows for 'get' in the Access field, 
+as these are the supported metrics that can be plotted.
+
+For example, let's command the module to a 3V drive voltage using the Control Panel in the Voltage Input tab and plot the *drive_volts* from the Brushless Drive client.
+
+.. note::
+    Please refer to the :ref:`Velocity and Voltage Based Control Mechanisms <manual_velocity_control_mechanisms>` documentation to learn more about Velocity and Voltage Control.
+
+.. warning::
+    Please remove all propellers from any module you plan on testing. Failure to do so can result in harm to you or others around you. Further, please ensure that your module is secured to a stationary platform or surface before attempting to spin it. 
+
+Set the 'Target Voltage' to 3.00 and click 'Send Command'. The module will ramp up to 3 volts over the next 5 seconds specified by the 'Ramp Time' value.
+After 5 seconds, the module should now have a constant, 3V, drive voltage. We can confirm this by plotting the drive volts.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/drive_volts_metric_plotting_example.png
+
+In order to visualize the module's drive voltage, navigate to the widget labeled 'Select a metric to plot'. The upper dropdown menu allows you to pick a client, and the lower a client entry.
+Please refer to :ref:`Getting Started with Vertiq's APIs <getting_started_with_apis>` to learn more about IQUART clients.
+
+In the upper dropdown menu, select *brushless_drive*. This corresponds to the :ref:`Brushless Drive <brushless_drive>` client.
+In the lower dropdown menu, select *drive_volts*, which corresponds with brushless drive's drive_volts entry.
+Now, click 'Update Plot'. 
+
+Notice that the *drive_volts* metric is added to the 'Active metrics' widget. By default, *obs_velocity*, *watts*, and *amps* are preloaded in the 'Active metrics' widget.
+Selecting a metric activates it, and updates the live plot with the now active metric. Only one metric can be active at a time, but there is no limit to the number of metrics that can be added to the 'Active metrics' widget.
+
```

Comment:
curiosity question again, not to do with docs...does VTT save the metrics I've added over time. For example, I add drive_volts now. Next time I open VTT, will drive_volts still be there?

Reply:
No it does not save, but that is a good feature to have. I will put it in the backlog